library/modules/Items.cpp

```diff
@@ -993,46 +1005,43 @@ static bool detachItem(MapExtras::MapCache &mc, df::item *item)
             case general_ref_type::CONTAINED_IN_ITEM:
                 if (auto item2 = ref->getItem())
                 {
-/* TODO: understand how this changes for v50
+                    /* TODO: understand how this changes for v50
                     // Viewscreens hold general_ref_contains_itemst pointers
                     for (auto screen = Core::getTopViewscreen(); screen; screen = screen->parent)
                     {
                         auto vsitem = strict_virtual_cast<df::viewscreen_itemst>(screen);
                         if (vsitem && vsitem->item == item2)
                             return false;
                     }
```

Comment:
this block of commented out code can be removed (viewscreen refs no longer exist and aren't coming back)